crates/solver/src/driver.rs

```diff
@@ -385,13 +385,15 @@ impl Driver {
             current_block::block_number(&self.block_stream.borrow())?;
 
         let before_count = auction.orders.len();
-        self.in_flight_orders.update_and_filter(&mut auction);
+        let inflight_order_uids = self.in_flight_orders.update_and_filter(&mut auction);
         if before_count != auction.orders.len() {
             tracing::debug!(
-                "reduced {} orders to {} because in flight at last seen block {}",
+                "reduced {} orders to {} because in flight at last seen block {}, order in flight: {:?}",
                 before_count,
                 auction.orders.len(),
-                auction.block
+                auction.block,
```

Comment:
I realized this should be:
```suggestion
                auction.latest_settlement_block,
```

Since we split off in-flight orders based on the last indexed settlement event, and not the block at the time the auction was being cut.